wintoast: add ARM64 support

dennisameling opened the pull request:
Needs https://github.com/git-for-windows/WinToast/pull/2 to be merged first

Visual Studio has ARM64 support since the 2017 edition. Let's add support for that architecture in G4W's implementation.

Signed-off-by: Dennis Ameling <dennis@dennisameling.com>

dennisameling commented on mingw-w64-wintoast/PKGBUILD:
```diff
@@ -12,7 +12,7 @@ license=('MIT')
 options=('!strip')
 
-source=("${_realname}"::"git+https://github.com/git-for-windows/WinToast#tag=g4w-20171117-01")
+source=("${_realname}"::"git+https://github.com/git-for-windows/WinToast#tag=TODO_WHEN_PR_MERGED")
```
This needs https://github.com/git-for-windows/WinToast/pull/2 to be merged first

dscho reviewed (approved):
I used the opportunity to update `wintoast` to use a better way to build: using `hMSBuild.bat`.

dennisameling commented:
@dscho thanks for the updates! I fixed one minor thing. Replaced

```
ARM64) WARCH=ARM64; OUTDIR=ARM64/Release;;
```

with

```
aarch64) WARCH=ARM64; OUTDIR=ARM64/Release;;
```

... and now the build works correctly. Here's the artifact in case you're interested:

[mingw-w64-clang-aarch64-wintoast-1.0.0.275.a44fa02-1-any.pkg.zip](https://github.com/git-for-windows/build-extra/files/9927976/mingw-w64-clang-aarch64-wintoast-1.0.0.275.a44fa02-1-any.pkg.zip)

Tested with the following command on Windows 11:

```
wintoast --appname "Git for Windows" \
--appid GitForWindows.Updater \
--image /mingw64/share/git/git-for-windows.ico \
--text "Download and install Git for Windows DEMO?" \
--action Yes --action No --expirems 15000
```

![image](https://user-images.githubusercontent.com/17739158/199712794-64844ca0-aa8a-4b32-8fda-77ca7dbc2475.png)

I'll come up with a separate (unrelated) PR in a few days to improve the logo quality and switch it to PNG, as discussed in the other PR.
